src/graph_spectrum.js

```diff
@@ -292,11 +292,12 @@ export function FlightLogAnalyser(flightLog, canvas, analyserCanvas) {
 
     this.importSpectrumFromCSV = function(files) {
       const maxImportCount = 5;
+      let importsLeft = maxImportCount - GraphSpectrumPlot.getImportedSpectrumCount();
+
       for (const file of files) {
-        if (GraphSpectrumPlot.getImportedSpectrumCount() == maxImportCount) {
+        if ((importsLeft--) == 0) {
```

Comment:
Prefix operator take precedence over assignment operator.
```suggestion
        if (importsLeft-- == 0) {
```

Reply:
Yes, i know about. But this is for "good reading"...